shell/common/dart_native_benchmarks.cc

```diff
@@ -0,0 +1,68 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/common/shell.h"
+
+#include "flutter/benchmarking/benchmarking.h"
+#include "flutter/shell/common/thread_host.h"
+#include "flutter/testing/dart_isolate_runner.h"
+#include "flutter/testing/fixtures_base.h"
+#include "flutter/testing/testing.h"
+#include "runtime/dart_vm_lifecycle.h"
+
+namespace flutter::testing {
+
+class DartNativeBenchmarks : public FixturesBase, public benchmark::Fixture {
+ public:
+  DartNativeBenchmarks() : FixturesBase() {}
+
+  void SetUp(const ::benchmark::State& state) {}
+
+  void TearDown(const ::benchmark::State& state) {}
+
+  void Wait() { latch_.Wait(); }
```

Comment:
I am not sure a latch needs to be part of the fixture. Can the benchmark just not create one on its own as needed? That it is an auto reset latch is not immediately obvious from using the fixture. The other setup as part of inheriting from FixturesBase is fine.

Reply:
Done.